### DISCUSSION TITLE
Plugin was not loaded: required plugin "com.intellij.modules.lang" not installed.

### DISCUSSION THREAD

**Discussion by Ashutosh-pixel** [consulo/consulo#459] [Q&A] [answered]
![pic](https://user-images.githubusercontent.com/66782335/110737612-b37ad480-8253-11eb-9495-a7b14fc4e3f9.png)
I have getting error while installing plugins from JetBrains market place.

Error: Fail to load plugin descriptor from file <plugin name>.jar
Error : Plugin was not loaded: required plugin "com.intellij.modules.lang" not installed.

**Comment by VISTALL** [comment, marked as answer]
Hello. IDEA plugins is not supported without modifications. Also depends which API is using, due some IDEA API is not provided by Consulo.

Thanks
